wit-manifest.json

```diff
@@ -1,7 +1,7 @@
 [
     {
-        "commit": "9961c82d62797c38c9dbd4c3f55ae1195df44ff5",
+        "commit": "a07f2ae4e30259c6ff3b9f0f3cd08baabb4c1503",
         "name": "soc-freedom-sifive",
-        "source": "git@github.com:sifive/soc-freedom-sifive.git"
+        "source": "git@github.com:zizztux/soc-freedom-sifive.git"
```

Comment:
This is just a reminder to restore this back to the upstream soc-freedom-sifive Git repository once the upstream changes are merged in.